### PR TITLE
dist/Locale-Maketext - synch Makefile.PL from CPAN

### PR DESCRIPTION
**Pull request by demerphq**
Fixes this module for https://github.com/Perl/perl5/issues/20874

### CODE REVIEW

**Comment by toddr** [comment]
@demerphq is there a reason we're wanting to pull this file in?

**Comment by haarg** [comment]
See https://github.com/Perl/perl5/pull/20867#issuecomment-1448422564 for the reason I want to do this for everything in `dist`.

**Comment by demerphq** [comment]
> See https://github.com/Perl/perl5/pull/20867#issuecomment-1448422564 for the reason I want to do this for everything in dist.

And I second that sentiment. I am going to create an issue for this in fact.

File: dist/Locale-Maketext/Makefile.PL [hunk 1]
```diff
@@ -0,0 +1,45 @@
+require 5.004;
```
Comment (tonycoz) [inline dist/Locale-Maketext/Makefile.PL]: The module uses `our` and `no warnings;`, so this should be 5.006.

Comment (demerphq) [reply dist/Locale-Maketext/Makefile.PL]: Fixed, also made the INSTALLDIRS similar to the other examples you provided.

**Comment by demerphq** [comment]
@tonycoz - is this ok?

**Review by tonycoz — APPROVED** [review]

**Pull request merged**